veldymanov/tile-shop-be
# Task 6

website: https://d2lvjuwl8mt9u1.cloudfront.net/
frontend PR:  https://github.com/veldymanov/tile-shop-fe/pull/7

Tasks:
6.1 - Done
6.2 - Done
6.3 - Done
**createProductTopic** unit testing - not yet
Filter Policy for SNS topic - Done

Testing:
1. Go to https://d2lvjuwl8mt9u1.cloudfront.net/admin/products.
2. Use testing csv files from: https://github.com/veldymanov/tile-shop-be/tree/task-6/testing-files.
3. Update testing file (if add new 'sku' - new product will be created;  if change in existing product ' title', 'description', 'price', or 'count' - these fields will be updated).
4. Reload website to see changes in DB.
